src/components/Modals/TeleportModal.vue

```diff
@@ -178,6 +178,9 @@ export default defineComponent({
     min-height: 0;
     margin-left: auto;
     margin-right: auto;
+    display: flex;
+    align-items: center;
+    pointer-events: none;
```

Comment:
This might have influenced how the Reset Requirement modal looks. Also the Reset requirement modal doesn't seem to reset.

https://user-images.githubusercontent.com/54298311/118169063-833be780-b3f6-11eb-9afd-fd4056c929db.mov

Reply:
Other modals look ok to me